pinot-core/src/main/java/org/apache/pinot/core/segment/processing/genericrow/GenericRowFileWriter.java

```diff
@@ -66,7 +65,10 @@ public void write(GenericRow genericRow)
   @Override
   public void close()
       throws IOException {
-    _offsetStream.close();
-    _dataStream.close();
+    try {
```

Comment:
@Jackie-Jiang @swaminathanmanish what's the preferred way to handle this? The current Manish's code is more elegant way of doing sth like the following? When I looked up online, I get some answers like the following:

```
public void close() {
  try {
    _offsetStream.close()
  } catch (IOException) {
    // logging error
  }

  try {
    _datastream.close()
  } catch (IOException e) {
    // log error
  }
}

```

Reply:
@snleee IMO your code is more clear, but it won't throw the exception out

Reply:
Yes, we need the exception, but at the same time close resources. I prefer the current way, unless you strongly feel otherwise.